changelogs/fragments/3726-terraform-missing-parameters-planned-fix.yml

```
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - terraform - fix command options being ignored during planned/plan in function ``build_plan`` such as lock or ``lock_timeout``
```

You do mean the `lock` parameter, right? In that case it should also be marked appropriately:
```suggestion
  - terraform - fix command options being ignored during planned/plan in function ``build_plan`` such as ``lock`` or ``lock_timeout``
```

---

Yes sorry, i missed to update it.